programs/zk-token-proof/src/lib.rs

```diff
@@ -32,6 +32,8 @@ pub const VERIFY_GROUPED_CIPHERTEXT_2_HANDLES_VALIDITY_COMPUTE_UNITS: u64 = 6_40
 pub const VERIFY_BATCHED_GROUPED_CIPHERTEXT_2_HANDLES_VALIDITY_COMPUTE_UNITS: u64 = 13_000;
 pub const VERIFY_FEE_SIGMA_COMPUTE_UNITS: u64 = 6_500;
 
+const PROOF_OFFSET_LENGTH: usize = 5;
```

Comment:
nit to maybe make this clearer?
```suggestion
const INSTRUCTION_DATA_LENGTH_WITH_PROOF_ACCOUNT: usize = 5;
```